code/modules/surgery/generic.dm

```diff
@@ -52,6 +52,7 @@
 	"<span class='notice'> You have made an incision on [target]'s [affected.name] with \the [tool].</span>",)
 	affected.open = 1
 	affected.status |= ORGAN_BLEEDING
+	target.bad_external_organs += affected
```

Comment:
I'm not sure how well this is going to work or if this is even using the list properly, to be honest.

Wound code is horrible and should be purged in hellfire.